test/unittests/execute_test.cpp

```diff
@@ -436,7 +436,7 @@ TEST(execute, i64_load_all_variants)
 
     for (const auto& test_case : test_cases)
```

Comment:
If you changed the other loop to `for (const auto& [instr, expected] : test_cases)` why not do it here too?

Btw, shouldn't we just get all this refactoring stuff merged separately?